From c85038dd137fd93cc6fe23e2fde1ab8b5fe9f6bf Mon Sep 17 00:00:00 2001
From: Julien Castelain <julien.castelain@liferay.com>
Date: Tue, 14 May 2019 10:47:25 +0200
Subject: [PATCH] LPS-89596 Cannot Drag Image from top content line in IE11
 using Alloy Editor

Changed position of drag icon for IE.
---
 plugins/widget/plugin.js | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/plugins/widget/plugin.js b/plugins/widget/plugin.js
index 5eb6d93512..65e0700bd0 100644
--- a/plugins/widget/plugin.js
+++ b/plugins/widget/plugin.js
@@ -1607,6 +1607,10 @@
 					y: domElement.offsetTop - DRAG_HANDLER_SIZE
 				};
 
+			if ( CKEDITOR.env.ie ) {
+				newPos.y = 0;
+			}
+
 			if ( oldPos && newPos.x == oldPos.x && newPos.y == oldPos.y )
 				return;
 
